dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -349,6 +358,42 @@ class CompileTest {
 
     return _kSdkNameToMetricNameMapping[sdkName];
   }
+
+  static Future<Map<String, dynamic>> getSizesFromApk(String apkPath) async {
+    final  String output = await eval('unzip', <String>['-v', apkPath]);
+    final List<String> lines = output.split('\n');
+
+    final Map<String, _UnzipListEntry> fileToMetadata = <String, _UnzipListEntry>{};
+
+    // First three lines are header, last two lines are footer.
+    for (int i = 3; i < lines.length - 2; i++) {
+      final List<String> line = lines[i].trim().split(new RegExp('\\s+'));
```

Comment:
nit: the parsing logic is split between this loop and the `_UnzipListEntry` constructor. I'd move everything into the constructor.

Reply:
Good call! Done.